Travel Scheduler/View Controllers/HomeCollectionViewController.m

```diff
@@ -266,22 +341,24 @@ - (void)makeSchedule
     if(self.arrayOfSelectedPlaces.count > 0) {
         [[[[self.tabBarController tabBar]items]objectAtIndex:1]setEnabled:TRUE];
         ScheduleViewController *destView = (ScheduleViewController *)[[self.tabBarController.viewControllers objectAtIndex:1] topViewController];
-        bool isFirstSchedule = NO;
-        if(destView.selectedPlacesArray == nil) {
-            isFirstSchedule = YES;
-        }
         destView.selectedPlacesArray = self.arrayOfSelectedPlaces;
         destView.regenerateEntireSchedule = true;
         destView.home = self.home ? self.home : self.hub;
         destView.hub = self.hub;
-        if(!isFirstSchedule) {
+        if(destView.selectedPlacesArray != nil) {
```

Comment:
I don't think this is the same thing as what I was trying to do. If you put this here, destView.selectedPlacesArray is always not nil because you already set the selectedPlacesArray to equal self.arrayOfSelectedPlaces so there's no point in having an if statement. I was trying to check if the destView.selectedPlacesArray is nil before we set it to self.arrayOfSelectedPlaces so I think you still need to keep the deleted lines. (Just wondering though, does it work? I tried removing the if statement in general and the formatting of the schedule view controller the first time was messed up).